elasticdl/python/common/k8s_client.py

```diff
@@ -367,19 +385,23 @@ def _create_ps_worker_pod(self, pod_name, type_key, index_key, **kargs):
     def create_worker(self, **kargs):
         pod_name = self.get_worker_pod_name(kargs["worker_id"])
         return self._create_ps_worker_pod(
-            pod_name, "worker", kargs["worker_id"], **kargs
+            pod_name, "worker", kargs["worker_id"], expose_ports=True, **kargs
```

Comment:
For worker pods, `expose_ports=True` only in DistributionStrategy.ALLREDUCE, not in DistributionStrategy.PARAMETER_SERVER.

Reply:
Thanks. Addressed.